src/ros_commo.py

```diff
@@ -169,7 +169,7 @@ def glance_at(self, face_id):
 	# Turn only the eyes towards the given target point.
 	# Coordinates: meters; x==forward, y==to Eva's left.
 	def gaze_at_point(self, x, y, z):
-		print "gaze at point: ", x, y, z
+		#print "gaze at point: ", x, y, z
 
```

Comment:
It just publishes a lot of stuff too fast and we lose important messages required while debugging. I can re - enable if you would like me to